### ISSUE TITLE
A11Y: various issues

### ISSUE THREAD

**Issue by andrejkaPry** [missive/emoji-mart#910]
Hi there, we stumbled upon various issues regarding a11y. Could you please navigate these?

1. Category tabs:
- currently possible to navigate only via Tab key, not arrows.
- not properly announced for screen readers, e.g.: 'Smileys and People, selected, tab, 1 of 8' -> incorrect label name

2. Skin Tone selection:
- not announcing skin tone selected, only role are announced as 'Choose skin tone, button'
- after choosing the skin tone, focus disappears

3. Emoji navigation:
- from Search input is possible to navigate only via arrows not Tab key

4. Emoji picker cannot be dismissed by Esc key - need to press few times.

**Comment by andrejkaPry** [comment]
First two already carry [this fix](https://github.com/missive/emoji-mart/pull/916), if you'd approve we could merge.

**Comment by PatrickJoannisse** [comment]
We had a WCAG audit recently and found those issues are also impacting us.

**Comment by andrejkaPry** [comment]
Hey @PatrickJoannisse, we already forked and fixed the issues as we had to move forward with this. https://github.com/slidoapp/emoji-mart 
It also is already published on https://www.npmjs.com/package/@slidoapp/emoji-mart, https://www.npmjs.com/package/@slidoapp/emoji-mart-react, https://www.npmjs.com/package/@slidoapp/emoji-mart-data. Feel free to use it :)
